mobile/lib/profile/screens/profile_page.dart

```diff
@@ -0,0 +1,217 @@
+                                CustomCounter(
+                                    text: profilePageData.finishedActivitiesCount!.toString()
+                                ),
+                                const CustomCounterDescription(text: 'Aktualny daily streak'),
+                                CustomCounter(
+                                    text: profilePageData.dailyStreak.toString()
+                                ),
+                                const CustomCounterDescription(text: 'Liczba znajomych'),
+                                CustomCounter(
+                                    text: profilePageData.friendsCount.toString()
+                                )
+                              ],
+                            ),
+                          ),
+                        ),
+                        Padding(
+                            padding: EdgeInsets.symmetric(
+                                horizontal: screenWidth * 0.1,
+                                vertical: screenHeight * 0.02),
+                            child: SizedBox(
+                              width: screenWidth * 0.8,
+                              height: screenHeight * 0.1,
+                              child: ElevatedButton(
+                                onPressed: null,
+                                style: ButtonStyle(
+                                    backgroundColor:
+                                        MaterialStateProperty.all<Color>(
+                                            Colors.grey[200]!),
+                                    shape: MaterialStateProperty.all<
+                                            RoundedRectangleBorder>(
+                                        RoundedRectangleBorder(
+                                            borderRadius:
+                                                BorderRadius.circular(15),
+                                            side: const BorderSide(
+                                                color: Colors.grey)))),
+                                child: const Center(
+                                  child: Text(
+                                    'Znajomi',
+                                    style: TextStyle(
+                                        fontSize: 34,
+                                        color: Colors.deepPurpleAccent,
+                                        fontWeight: FontWeight.w300),
+                                  ),
+                                ),
+                              ),
+                            )),
+                      ],
+                    ),
+                  ),
+                ),
+                const CustomNavigationBar(),
+              ],
+            );
+          } else {
+            return const Text('Pobieranie danych');
+          }
+        },
+      ),
+    )));
+  }
+}
+
+class ProfilePageData {
```

Comment:
do oddzielnego pliku, w pewnym sensie to takiego dto jest

Reply:
zrobiłem tak, ale wtedy nie mogę skorzystać z tej klasy w FutureBuilder

Reply:
ja jak wyrzuciłem tą klasę profilepagedata do oddzielnego pliku i ją zaimportowałem, to apka działa git :smiley:

Reply:
faktycznie, też u mnie działa